bioptim-zero-to-hero-Rototrans.ipynb

```diff
@@ -0,0 +1,367 @@
+{
```

Comment:
<p>"This tutorial is a trivial example on how to manage Rototranslation object with biorbd.</p><p>It is designed to show how one can find the rototranslation matrix between any segment.</p><p>The rototranslation Tji (aka homogeneous transform) allows to transform any point P in frame i to another frame j."</p><p> </p><p>Can you add a the symbolic matrix in the text which describes its components such as <a href="https://slideplayer.com/slide/5160476/16/images/13/Quiz+2+Geometric+Interpretation+Inverse+Homogeneous+Matrix.jpg" target="_blank">here</a>. Ask me if you're lost.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ipuch/bioptim-zero-to-hero/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>